### PR TITLE
feat: Add a build for mediawiki 1.39

### PR DESCRIPTION
**Pull request by kwisatz**

### CODE REVIEW

File: mw39/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,153 @@
+# Copy of https://github.com/wikimedia/mediawiki-docker/edit/master/1.36/apache/Dockerfile with PHP and MW versions changed
+FROM php:8.0-apache as mw39
```
Comment (JeroenDeDauw) [inline mw39/Dockerfile]: Do you really need 8.0? I recommend switching to 8.1 when using MW 39. We are running MW + WB + WB extensions on PHP 8.1, so this is likely fine for you as well.

Comment (kwisatz) [reply mw39/Dockerfile]: Euh… no, and choosing 8.1 from the get-go, I would probably have saved myself quite some headaches with composer not respecting platform configuration, etc…

Comment (kwisatz) [reply mw39/Dockerfile]: Updated the `Dockerfile` and the `README`

Comment (kwisatz) [reply mw39/Dockerfile]: I advise against merging this just yet. I'm seeing some very weird behavior with the Mediawiki installation where it claims there is no `Localsettings.php` file. May not be related to this image, but let me make sure it isn't.

Comment (kwisatz) [reply mw39/Dockerfile]: I have found that the problem seems to be the sticky bit here on the `html` directory:

```bash
root@0df905e2e6f9:/var/www/html# ls -ld
drwxrwxrwt 1 www-data www-data 4096 Mar 28 09:55 .
```

Removing it makes this work.

I doubt that changing the `PHP` version is responsible for that, so I need to check where this is coming from.

Comment (kwisatz) [reply mw39/Dockerfile]: I'm having the same problem on 8.0 now… and while I don't know where this is coming from exactly, I don't think it has anything to do directly with the image, so feel free to merge if you want.

**Comment by JeroenDeDauw** [comment]
I will build the images later today

**Comment by JeroenDeDauw** [comment]
They are building and uploading now. So will be live within the hour.

I've also restored the PHP version in the MW 35 image.

**Pull request merged**